morpheushelper/cogs/mod.py

```diff
@@ -242,11 +242,11 @@ async def roles_mute(self, ctx: Context, role: Role):
 
     @commands.command()
     @guild_only()
-    async def report(self, ctx: Context, member: Member, *, reason: str):
+    async def report(self, ctx: Context, member: Union[Member, int], *, reason: str):
```

Comment:
Please add `User` to the member type Union so that also Users that left the server can be used for the command.